www/src/css/_home.scss

```diff
@@ -0,0 +1,90 @@
+@import './_var';
+
+.home {
+  .grid-body {
+    padding-top: 0;
+  }
+
+  .grid-body-header {
+    background: #2e5e82;
+    border-top: solid 5px #ffffff;
```

Comment:
I think it looks better without the border top, especially since it's sticky you scroll down 5px and no more border anyway :)